src/commands/utils/wolframalpha.ts

```diff
@@ -1,82 +1,80 @@
 import { URLSearchParams } from 'url';
-import request from 'request';
+import fetch from 'node-fetch';
 import { CommandDefinition } from '../../lib/command';
 import { makeEmbed, makeLines } from '../../lib/embed';
-import { CommandCategory, Channels } from '../../constants';
+import { CommandCategory } from '../../constants';
 import Logger from '../../lib/logger';
 
-const WOFLRAMALPHA_API_URL = 'http://api.wolframalpha.com/v2/query?';
+const WOLFRAMALPHA_API_URL = 'http://api.wolframalpha.com/v2/query?';
 const WOLFRAMALPHA_QUERY_URL = 'http://www.wolframalpha.com/input/?';
 
 export const wolframalpha: CommandDefinition = {
     name: ['wa', 'calc', 'ask'],
     description: 'Queries the Wolfram Alpha API',
     category: CommandCategory.UTILS,
     executor: async (msg) => {
-        const allowedChannel = Channels.BOT_COMMANDS;
-        if (msg.channelId === allowedChannel) {
-            const re = /\.\S+\s(.+)/;
-            let query: string = '';
-            try {
-                // eslint-disable-next-line prefer-destructuring
-                query = msg.content.match(re)[1];
-            } catch (e) {
-                Logger.error(e);
-            }
-            if (query.length <= 0) {
-                await msg.reply('Please enter a query.');
+        const splitUp = msg.content.replace(/\.wa\s+/, ' ').split(' ');
+
+        if (splitUp.length <= 1) {
+            await msg.reply('Please provide a query. For example: .wa "2 + 2"');
```

Comment:
```suggestion
            await msg.reply('Please provide a query. For example: `.wa 2 + 2`');
```

Preferred formatting

Reply:
Done it this way to demonstrate you do not need to use quotations when using the command, could add if necessary for documentation

Reply:
What do you mean? The way you currently have it implies you need to wrap your query with `" "`

Reply:
Silly me yeah I'll fix it.